aiohttp/multidict.py

```diff
@@ -305,6 +305,9 @@ def __init__(self, items):
     def __len__(self):
         return len(self._items)
 
+    def __repr__(self):
```

Comment:
yes, for sure)
I must say that `test_muldict.py` is almost as complicated as aiohttp parsers...
Can you point me where should I put tests for cython's multidict?